File: pset1/mario/less/mario.c
#include<cs50.h>
#include<stdio.h>


int main(void)
{
    int row;
    //user input
    do
    {
        row = get_int("Height: ");

    }
    while (row < 0 || row > 23);

    for (int i = 0; i < row; i++)
    {
        //for spaces
        for (int sp = 0; sp < row - i - 1; sp++)
        {
            printf(" ");
        }
        //for hashes
        for (int j = 0; j < i + 2; j++)
        {
            printf("#");
        }
        printf("\n");
    }

}

File: pset6/sentimental/mario/less/mario.py
from cs50 import get_int

row = get_int('Height: ')
while True:
    if row < 0 or row > 23:
        row = get_int('Height: ')
    else:
        break

for i in range(0, row):
    for sp in range(0, row-i-1):
        print('', end=' ')
    for j in range(0, i+2):
        print('#', end='')
    print('')
